app/src/main/java/org/lineageos/tv/launcher/SystemOptionsActivity.kt

```diff
@@ -226,8 +236,8 @@ class SystemOptionsActivity : ModalActivity(R.layout.activity_system_options),
             }
         }
 
-    override fun onItemClick(position: Int, sbn: StatusBarNotification) {
-        val notification = sbn.notification
+    override fun onItemClick(view: View, position: Int) {
+        val notification = (view as NotificationItem).statusBarNotification?.notification ?: return
```

Comment:
Yeah that's quite ugly, ListAdapter will force you to make proper ViewHolder abstractions, take a look at Twelve for examples (SimpleListAdapter.kt and any usage)